src/distributions/mod.rs

```diff
@@ -279,6 +279,8 @@ where
 /// *   Arrays (up to 32 elements): each element is generated sequentially;
 ///     see also [`Rng::fill`] which supports arbitrary array length for integer
 ///     types and tends to be faster for `u32` and smaller types.
+///     When using `rustc` ≥ 1.51, enable the `min_const_gen` feature to support
+///     arrays larger than 32 elements.
```

Comment:
Can we maybe add the following here (after your addition)?

> Note that [`Rng::fill`] and `Standard`'s array support are *not* equivalent: the former is optimised for integer types (using fewer RNG calls for element types smaller than the RNG word size), while the latter supports any element type supported by `Standard`.

Reply:
done